docs/whats-new/production-release-notes.md

```diff
@@ -1,5 +1,18 @@
 # Production release notes
 
+## April 2024
+
+**03 April 2024**
+
+
+Frontend version: 1.0.0-20240321.1254 | Backend version: 1.95.26-20240325.1331
+
+| Type | Change | Description |
+|------|--------|-------------|
+| **Feature** | Expose app roles management on TechPass portal | Tenants can now manage app roles (excluding app roles used by TechBiz) on the portal under the *edit application* page. |
+| **Change** | Autonomous app role assignments for TechBiz-integrated applications | Previously available only via Automation API, tenants can now autonomously manage app role assignments for TechBiz-integrated applications (excluding TechBiz group assignments) directly through the TechPass portal. |
```

Comment:
need to add the email feature for prod as well

Reply:
@yqyeoh updated